aspnet/publishing/index.rst

```diff
@@ -13,3 +13,4 @@ Publishing and Deployment
     how-to-customize
     iis-with-msdeploy
     web-publishing-vs
+	Hosting Web Apps in Docker <https://azure.microsoft.com/documentation/articles/vs-azure-tools-docker-hosting-web-apps-in-docker/>
```

Comment:
This docker doc appears to be a close duplicate of https://docs.asp.net/en/latest/publishing/docker.html. I don't think we need both. Which one should we keep?